source/index.html

```diff
@@ -212,7 +212,11 @@
     background-repeat: no-repeat;
 }
 .sidebar-separator { margin-bottom: 20px; }
-.sidebar-find-search { font-family: inherit; font-size: 13px; margin: 0px 20px 8px 20px; padding: 8px 16px 8px 16px; background: #fff; border-radius: 16px; border: 0; outline: 0; }
+.sidebar-find-box { position:relative; }
+.sidebar-find-search { position:relative; width:85%; font-family: inherit; font-size: 13px; margin: 0px 20px 8px 20px; padding: 8px 16px 8px 16px; background: #fff; border-radius: 16px; border: 0; outline: 0; }
+.siderbar-find-edge-icon { font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace; font-size: 12px; position:absolute; background-color:#0000; top:8px; right:40px; user-select: none; -webkit-user-select: none; -moz-user-select: none; }
```

Comment:
Instead of positioning manually using `top`, is there a way to center automatically? Is there a set of characters that is guaranteed to be aligned is all system fonts or should this change to use SVG icons? 

![image](https://github.com/lutzroeder/netron/assets/438516/32d84ae5-1c2a-4b7c-b8d3-bd624bc7fefe)

Reply:
I'm not really sure to be honest. The world of fonts looks really complicated. The unaligned positioning of the icons becomes rather obvious when the font-size gets bigger, so just as you mentioned, it's undesirable to position the icons manually.

![image](https://github.com/lutzroeder/netron/assets/1131125/668666d1-22f4-4fdd-b87e-9f356c00a4a5)